### PR TITLE
docs: DLT-2122 blog post for recipe renames

### PR DESCRIPTION
**Pull request by braddialpad**
Blog post for the recipe class renames.

### CODE REVIEW

File: packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js [hunk 1]
```diff
@@ -48,20 +48,20 @@ const _setChildWrappers = () => {
...
+  listItemsFormatBtn = wrapper.find('[data-qa="d-recipe-editor-list-items-btn"]');
+  addLinkBtn = wrapper.find('[data-qa="d-recipe-editor-add-link-btn"]');
+  alignLeftBtn = wrapper.find('[data-qa="d-recipe-editor-align-left-btn"]');
+  alignCenterBtn = wrapper.find('[data-qa="d-recipe-editor-align-center-btn"]');
+  alignRightBtn = wrapper.find('[data-qa="d-recipe-editor-align-right-btn"]');
+  alignJustifyBtn = wrapper.find('[data-qa="d-recipe-editor-align-justify-btn"]');
+  orderedListBtn = wrapper.find('[data-qa="d-recipe-editor-ordered-list-items-btn"]');
+  blockquoteBtn = wrapper.find('[data-qa="d-recipe-editor-blockquote-btn"]');
+  codeblockBtn = wrapper.find('[data-qa="d-recipe-editor-code-block-btn"]');
+  quickRepliesBtn = wrapper.find('[data-qa="d-recipe-editor-quick-replies-btn"]');
```
Comment (juliodialpad) [inline packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js]: Don't think this is really necessary, I'll prefer to keep all data-qa with `dt-` prefix so it's easier to look for it when working on tests.

Also you're looking for a component part not an actual class, so it makes sense to keep it as `dt-*`

**Review by juliodialpad — COMMENTED** [review]
Looks ok to me, my only concern is with the data-qa attributes. If we want to move them to `d-` it is ok for me but I'll prefer to keep them as `dt-` as they're also referencing the component name.

File: packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js (hunk 1 above)
Comment (juliodialpad) [reply packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js]: Think this review belongs to #581 didn't noticed that you've merged that branch here.

File: apps/dialtone-documentation/docs/about/whats-new/posts/2024-12-10.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+heading: Recipe class renames (POSSIBLE BREAKING CHANGE)
+author: Brad Paugh
+posted: '2024-12-10'
+---
+
+<BlogPost :author="$frontmatter.author" :posted="parse($frontmatter.posted, 'y-M-d', new Date())" :heading="$frontmatter.heading">
+
+Hi again, similar to our previous component class renames, we've now renamed some classes in our recipes to be consistent with the naming of the rest of our components. Similar to last time this will only be a breaking change if you are manually referencing these internal classes in your application. The fix is the same as last time, just find / replace the old class name to the new class name in your consuming application.
+
+Note that this will also include all sub elements of these classes, so if it says to rename `dt-list-item` to `d-list-item` this will also be the case for `dt-list-item--focusable`
```
Comment (juliodialpad) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2024-12-10.md]: change this one for an example with a recipe please

**Comment by braddialpad** [comment]
Sorry I had the base branch set incorrectly. This PR should now be only the blog post.

File: packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js (hunk 1 above)
Comment (braddialpad) [reply packages/dialtone-vue2/recipes/conversation_view/editor/editor.test.js]: yeah.. I was not sure whether we should change these ones or not. I think leaving them as dt is fine. I will make the change.

File: apps/dialtone-documentation/docs/about/whats-new/posts/2024-12-10.md (hunk 2 above)
Comment (braddialpad) [reply apps/dialtone-documentation/docs/about/whats-new/posts/2024-12-10.md]: Updated.

**Review by juliodialpad — APPROVED** [review]
Thanks!

**Pull request merged**